test/unit/spec/MfaVerify_spec.js

```diff
@@ -2301,6 +2305,76 @@ function (Okta,
                 });
               });
             });
+            itp('starts poll after a delay of 6000ms', function () {
+              return setupOktaPush().then(function (test) {
+                spyOn(LoginUtil, 'callAfterTimeout').and.callFake(function() {
+                  // reducing the timeout to 100 so that test is fast.
+                  return setTimeout(arguments[0], 100);
+                });
+                $.ajax.calls.reset();
+                test.setNextResponse(resChallengePush);
+                test.form[0].submit();
+                return tick(test).then(function() {
```

Comment:
we should use, if possible, some `Expect.waitFor*` instead of `tick`. That is preferred since it is more deterministic. Maybe wait for button to be disabled or an spy being called
Applies for all the new tests